Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Staging/Subsequences/Configurator.mo

```diff
@@ -144,52 +144,52 @@ protected
   connect(pro.y,matMax. u)
-    annotation (Line(points={{-119,-90},{-102,-90}}, color={0,0,127}));
+    annotation (Line(points={{-118,-80},{-102,-80}}, color={0,0,127}));
   connect(matMax.y,reaToInt. u)
-    annotation (Line(points={{-79,-90},{-62,-90}}, color={0,0,127}));
-  connect(reaToInt.y, yTyp) annotation (Line(points={{-39,-90},{90,-90},{90,-80},
-          {230,-80}}, color={255,127,0}));
-  connect(reaToInt.y, intToRea1.u) annotation (Line(points={{-39,-90},{-28,-90},
+    annotation (Line(points={{-78,-80},{-62,-80}}, color={0,0,127}));
+  connect(reaToInt.y, yTyp) annotation (Line(points={{-38,-80},{240,-80}},
+                      color={255,127,0}));
+  connect(reaToInt.y, intToRea1.u) annotation (Line(points={{-38,-80},{-28,-80},
           {-28,-130},{-22,-130}},color={255,127,0}));
   connect(intToRea1.y, sort.u)
-    annotation (Line(points={{1,-130},{18,-130}},  color={0,0,127}));
+    annotation (Line(points={{2,-130},{18,-130}},  color={0,0,127}));
   connect(sort.y, reaToInt1.u)
-    annotation (Line(points={{41,-130},{58,-130}},   color={0,0,127}));
-  connect(reaToInt.y,intEqu. u1) annotation (Line(points={{-39,-90},{90,-90},{90,
+    annotation (Line(points={{42,-130},{58,-130}},   color={0,0,127}));
+  connect(reaToInt.y,intEqu. u1) annotation (Line(points={{-38,-80},{90,-80},{90,
           -110},{98,-110}},      color={255,127,0}));
-  connect(reaToInt1.y,intEqu. u2) annotation (Line(points={{81,-130},{90,-130},{
+  connect(reaToInt1.y,intEqu. u2) annotation (Line(points={{82,-130},{90,-130},{
           90,-118},{98,-118}},    color={255,127,0}));
   connect(mulAnd.y, assMes.u)
-    annotation (Line(points={{161.7,-110},{178,-110}},color={255,0,255}));
-  connect(intEqu.y, mulAnd.u) annotation (Line(points={{121,-110},{138,-110}},
+    annotation (Line(points={{162,-110},{178,-110}},  color={255,0,255}));
+  connect(intEqu.y, mulAnd.u) annotation (Line(points={{122,-110},{138,-110}},
     color={255,0,255}));
-  connect(staType.y, pro.u1) annotation (Line(points={{-179,-70},{-160,-70},{-160,
-          -84},{-142,-84}},color={0,0,127}));
-  connect(staDesCaps.y, yDesCap) annotation (Line(points={{-119,150},{100,150},{
-          100,60},{230,60}}, color={0,0,127}));
-  connect(staMinCaps.y, yMinCap) annotation (Line(points={{-119,110},{80,110},{80,
-          20},{230,20}}, color={0,0,127}));
+  connect(staType.y, pro.u1) annotation (Line(points={{-178,-60},{-160,-60},{-160,
+          -74},{-142,-74}},color={0,0,127}));
+  connect(staDesCaps.y, yDesCap) annotation (Line(points={{-118,150},{100,150},{
+          100,60},{240,60}}, color={0,0,127}));
+  connect(staMinCaps.y, yMinCap) annotation (Line(points={{-118,110},{80,110},{80,
+          20},{240,20}}, color={0,0,127}));
   connect(oneVec.y, sumNumChi.u)
-    annotation (Line(points={{-179,60},{-142,60}}, color={0,0,127}));
+    annotation (Line(points={{-178,60},{-142,60}}, color={0,0,127}));
   annotation (defaultComponentName = "conf",
```

Comment:
in the info section, the section number are different between difference version of RP1711. For later version (March and July versions):
- section 5.2.4.9 should be 5.2.4.12
- section 5.2.4.10 should be 5.2.4.13
- there is no design stage capacity anymore in July version, section 3.1.1.4, 2

Reply:
Inroduced the first two points. For the third, they still in the July version use "design capacity of Chiller X", so I'll use "design stage capacity", which is the sum of all design capacities of chillers in that stage.

Reply:
I double-checked the "stage design capacity" construct, and confirmed with version July, 5.2.4.5. wording.